pkg/kubelet/metrics/metrics.go

```diff
@@ -127,10 +127,21 @@ const (
 	// Metric for tracking garbage collected images
 	ImageGarbageCollectedTotalKey = "image_garbage_collected_total"
 
+	// Metric for tracking aligment of compute resources
+	ContainerAlignedComputeResourcesNameKey          = "container_aligned_compute_resources_count"
+	ContainerAlignedComputeResourcesScopeLabelKey    = "scope"
+	ContainerAlignedComputeResourcesBoundaryLabelKey = "boundary"
+
 	// Values used in metric labels
 	Container          = "container"
 	InitContainer      = "init_container"
 	EphemeralContainer = "ephemeral_container"
+
+	AlignScopePod       = "pod"
+	AlignScopeContainer = "container"
+
+	AlignedPhysicalCPU = "physical_cpu"
+	AlignedNUMAZone    = "numa_zone"
```

Comment:
do we refer to numa node as zone in other places? Is zone the same as node?

Reply:
AFAIK "numa zones" "numa cells" and "numa nodes" are all synonyms, and I'd like to avoid the overinflaction of the "node" term.

Reply:
Do we ever use NUMA zone term? I see only numa nodes for example here: https://kubernetes.io/docs/tasks/administer-cluster/memory-manager/ Will it be more confusing to mix terms interchangeably than we gain by not mixing "nodes" and 
"NUMA nodes"?

Reply:
ok, changing to "NUMA nodes"